main.go

```diff
@@ -79,39 +88,53 @@ func main() {
 
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
 
-	kubeConfig := ctrl.GetConfigOrDie()
-	kubeConfig.QPS = float32(qps)
-	kubeConfig.Burst = burst
-
 	if err := utilfeature.DefaultMutableFeatureGate.Set(featureGates); err != nil {
 		setupLog.Error(err, "Unable to set flag gates for known features")
 		os.Exit(1)
 	}
 
-	mgr, err := ctrl.NewManager(kubeConfig, ctrl.Options{
-		Scheme: scheme,
-		Metrics: server.Options{
-			BindAddress: metricsAddr,
-		},
-		WebhookServer: webhook.NewServer(
-			webhook.Options{
-				Port: 9443,
-			}),
-		HealthProbeBindAddress: probeAddr,
-		LeaderElection:         enableLeaderElection,
-		LeaderElectionID:       "6d4f6a47.x-k8s.io",
-		// LeaderElectionReleaseOnCancel defines if the leader should step down voluntarily
-		// when the Manager ends. This requires the binary to immediately end when the
-		// Manager is stopped, otherwise, this setting is unsafe. Setting this significantly
-		// speeds up voluntary leader transitions as the new leader don't have to wait
-		// LeaseDuration time first.
-		//
-		// In the default scaffold provided, the program ends immediately after
-		// the manager stops, so would be fine to enable this option. However,
-		// if you are doing or is intended to do any operation such as perform cleanups
-		// after the manager stops then its usage might be unsafe.
-		// LeaderElectionReleaseOnCancel: true,
-	})
+	var options manager.Options
+
+	kubeConfig := ctrl.GetConfigOrDie()
+	if configFile == "" {
+		options = ctrl.Options{
+			Scheme: scheme,
+			Metrics: server.Options{
+				BindAddress: metricsAddr,
+			},
+			WebhookServer: webhook.NewServer(
+				webhook.Options{
+					Port: 9443,
+				}),
+			HealthProbeBindAddress: probeAddr,
+			LeaderElection:         enableLeaderElection,
+			LeaderElectionID:       "6d4f6a47.x-k8s.io",
+			// LeaderElectionReleaseOnCancel defines if the leader should step down voluntarily
+			// when the Manager ends. This requires the binary to immediately end when the
+			// Manager is stopped, otherwise, this setting is unsafe. Setting this significantly
+			// speeds up voluntary leader transitions as the new leader don't have to wait
+			// LeaseDuration time first.
+			//
+			// In the default scaffold provided, the program ends immediately after
+			// the manager stops, so would be fine to enable this option. However,
+			// if you are doing or is intended to do any operation such as perform cleanups
+			// after the manager stops then its usage might be unsafe.
+			// LeaderElectionReleaseOnCancel: true,
+		}
+		kubeConfig.QPS = float32(qps)
+		kubeConfig.Burst = burst
+	} else {
+		opts, cfg, err := apply(configFile)
+		if err != nil {
+			setupLog.Error(err, "Unable to load the configuration")
+			os.Exit(1)
+		}
+		options = opts
+		kubeConfig.QPS = *cfg.ClientConnection.QPS
+		kubeConfig.Burst = int(*cfg.ClientConnection.Burst)
```

Comment:
If the user sets both the qps flag and the configFile flag, will the qps flag not take effect? ​​Have we considered this situation?

Reply:
It wouldn't take effect. Only configFile and featuregate flag is support in kueue currently. 
Considering supporting both configuration files and command-line arguments requires to add spf13/viper like dependencies.
I just want to keep it simple.

Reply:
Thanks for your reply. so, config is the first priority. Once configured, flag will not take effect.

Reply:
I add this comment to the description.